Entities/ACRocket.cpp

```diff
@@ -1023,11 +1023,15 @@ void ACRocket::Update()
     ////////////////////////////////////////
     // Hatch Operation
 
-    int lastFrame = m_FrameCount - 1;
+#if defined DEBUG_BUILD || defined MIN_DEBUG_BUILD
```

Comment:
I mention this elsewhere too, but I don't see any benefit to only having this assert show up in debug builds. See other comments below for more details.